src/me/john000708/barrels/Barrel.java

```diff
@@ -389,13 +417,14 @@ private boolean isSimiliar(ItemStack i1, ItemStack i2) {
 
         if (!itemStack2.hasItemMeta()) return false;
 
-        if (!itemStack2.getItemMeta().hasLore()) return false;
+        if (!Objects.requireNonNull(itemStack2.getItemMeta()).hasLore()) return false;
```

Comment:
Purposely throwing a NPE after the .hasItemMeta() check makes no sense whatsoever.